tests/test_metrics.py

```diff
@@ -122,7 +122,7 @@ def check_f05_metrics(metric_name, average_method):
     y_true = [1, 1, 1, 0, 0, 0]
     y_pred = [0, 1, 1, 1, 0, 0]
     skll_value = use_score_func(metric_name, y_true, y_pred)
-    sklearn_value = fbeta_score(y_true, y_pred, 0.5, average=average_method)
```

Comment:
@desilinguist , this is a change that addresses a warning error I got during local tests:
```FutureWarning: Pass beta=0.5 as keyword args. From version 0.25 passing these as positional arguments will result in an error```
Otherwise, there was nothing that sklearn shouted which was deprecation related.